perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -619,6 +619,10 @@ sub waste_container_actions {
     };
 }
 
+sub waste_staff_source {
+    return 3;
```

Comment:
Note master has a _set_user_source function that does this and more (9 for Officer, 3 for CSC), that might be good to use here.